Assets/Scripts/BirdScript.cs

```diff
@@ -0,0 +1,293 @@
+
+        if (Time.fixedTime == 22)
+            SaidToPlayer.Invoke("Crap, forgot about the droids.");
+
+    }
+
+    void PondUpdate()
+    {
+        if (Time.fixedTime == 1)
+            SaidToPlayer.Invoke("Hi again.");
+
+        if (Time.fixedTime == 3)
+            SaidToPlayer.Invoke("Welcome to the pond.");
+
+        if (Time.fixedTime == 6.5)
+            SaidToPlayer.Invoke("This is where I come to think.");
+
+        if (Time.fixedTime == 10)
+            SaidToPlayer.Invoke("I always find myself a bit more realaxed in a natural setting.");
+
+        if (Time.fixedTime == 14)
+            SaidToPlayer.Invoke("Even if it is simulated.");
+
+        if (Time.fixedTime == 16)
+            SaidToPlayer.Invoke("Like listening to rainstorms while you sleep.");
+
+        if (Time.fixedTime == 20)
+            SaidToPlayer.Invoke("Except I hate rain.");
+
+        if (Time.fixedTime == 22)
+            SaidToPlayer.Invoke("Because of the whole get wet deal.");
+
+        if (Time.fixedTime == 25)
+        {
+            WorkPanels.gameObject.SetActive(true);
+            SaidToPlayer.Invoke("Look around for a bit, I have some backstage work to get done here.");
+        }
+            
+
+        if (Time.fixedTime == 26)
+        {
+            eventStartTime = Time.fixedTime;
+            scriptLoaded = "Mumble";
+        }
+            
+    }
+
+    void OfficeUpdate()
+    {
+
+    }
+
+    void LightingUpdate()
+    {
+        if (Time.fixedTime == 1)
+            SaidToPlayer.Invoke("Hi, and welcome to...");
+
+        if (Time.fixedTime == 3)
+            SaidToPlayer.Invoke("Where'd you go?");
+
+        if (Time.fixedTime == 6.5)
+            SaidToPlayer.Invoke("Oh crap.");
+
+        if (Time.fixedTime == 10)
+            SaidToPlayer.Invoke("I think I just lost you.");
+
+        if (Time.fixedTime == 14)
+            SaidToPlayer.Invoke("Don't panic, just follow my voice.");
+
+        if (Time.fixedTime == 16)
+            SaidToPlayer.Invoke("I knew the random spawn location thing was a pain in the tail.");
+
+        if (Time.fixedTime == 20)
+            SaidToPlayer.Invoke("It's ok, i'm sure you can find me.");
+
+        if (Time.fixedTime == 22)
+            SaidToPlayer.Invoke("I hope.");
+    }
+
+
+    private void PlayerProceededTo(string pointMessage)
+    {
+        eventStartTime = Time.fixedTime;
+        if (pointMessage == "Player met Crow")
+            scriptLoaded = "FirstEncounter";
+
+    }
+
+    private void FoundCrowInLighting()
+    {
+        if (Time.fixedTime - eventStartTime == 0.5f)
+            SaidToPlayer.Invoke("Oh there you are.");
+    }
+
+    private void FirstEncounter()
+    {
+
+
+        if (Time.fixedTime - eventStartTime == 1f)
+        {
+            SaidToPlayer.Invoke("Oh...         ");
+            WorkPanels.gameObject.SetActive(false);
+        }
+            
+        if (Time.fixedTime - eventStartTime == 4)
```

Comment:
Just a suggestion. Instead of magic strings in these invokes. Maybe make a serialized list that can be added to?

Reply:
i will look into it for future iterations, need a better system for them